spec/lib/config_validator_spec.rb

```diff
@@ -15,9 +15,18 @@
         good_key => 'foo',
       }
```

Comment:
In order to test the stripping and downcasing, can you please add some more candidate keys, such as " YES" and "NO "?